### PR TITLE
[Icons]Revision of TechDraw Icons(lock)

### PR DESCRIPTION
**Pull request by Reqrefusion**
#14145

### CODE REVIEW

**Comment by Reqrefusion** [comment]
The lock icons here are clearly different from Skecher because their functions are completely different and therefore they are displayed in different ways.

**Review by kadet1090 — APPROVED** [review]

File: src/Mod/TechDraw/Gui/Resources/icons/TechDraw_ExtensionLockUnlockView.svg
Comment (maxwxyz) [inline src/Mod/TechDraw/Gui/Resources/icons/TechDraw_ExtensionLockUnlockView.svg:1]: Just delete the blue stroke on the keyholes and it's good.

**Review by maxwxyz — CHANGES_REQUESTED** [review]
Just one change.

**Review by maxwxyz — APPROVED** [review]

**Comment by maxwxyz** [comment]
@chennes can be merged

**Pull request merged**